startnode.sh

```diff
@@ -14,7 +13,7 @@ VALIDATOR_ADDR=$(noisd keys show validator -a)
 # rm -rf $HOME/.noisd
 rm -rf ~/.noisd/config/gentx
 # setup chain
-noisd init stargaze --chain-id localnet-1 --overwrite
+noisd init nois --chain-id localnet-1 --overwrite
```

Comment:
This is the moniker. What about calling it somehting like "local-validator-001" instead?